fonts/README.md

```diff
@@ -8,9 +8,9 @@ This directory contains the fonts used internally by MuseScore and the sources u
 
 * **bravura** The [_Bravura_](https://github.com/steinbergmedia/bravura/) fonts and the documentation for them. These fonts are **not** maintained by the MuseScore community, please **DO NOT EDIT** these fonts. 
 * **campania** The [_Campania_](https://github.com/MarcSabatella/Campania) font for Roman numeral analysis.  This font is not maintained by the MuseScore project, but it is open source and contributions from the MuseScore community are encouraged at the main site for the font.
-* **edwin** The _Edwin_ text font family. This font is maintained by the MuseScore project.
+* **edwin** The [_Edwin_](https://github.com/MuseScoreFonts/Edwin/) text font family. This is maintained by the MuseScore project.
 * **gootville** The [_Gonville_](http://www.chiark.greenend.org.uk/~sgtatham/gonville/) fonts for score musical symbols. This font is modified and maintained by the MuseScore project.
-* **leland** The _Leland_ font used in MuseScore scores for musical symbols. This is maintained by the MuseScore project.
+* **leland** The [_Leland_](https://github.com/MuseScoreFonts/Leland/) font used in MuseScore scores for musical symbols. This is maintained by the MuseScore project.
 * **mscore** The main font used in MuseScore scores for musical symbols (formerly known as [_Emmentaler_](http://lilypond.org/doc/v2.18/Documentation/notation/the-feta-font)), as well its counterpart used for texts.
```

Comment:
Why `mscore`, not just `Emmentaler`? Also it isn't the main font now.

Reply:
`mscore` is derived from `Emmentaler`, not identical. Internally it was always called `mscore`and still is, externally it was always called Èmmentaler`and still is. This Readme decribes the content od the subdirectories here.

Valid point about it not being the main font anymore.